style.css

```diff
@@ -0,0 +1,270 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  box-shadow: none;
+}
+
+body {
+  margin: 1rem;
+  padding: 1rem;
+  width: 327px;
+}
+
+.header_container {
+  padding: 0.5rem;
+  font-family: sans-serif;
+  display: flex;
+  align-items: center;
```

Comment:
- [ ] Please make sure you use the right `font family` specified in your chosen Figma design 🙏

Reply:
The font family specified in the figma design is not in css font family

Reply:
Hi @Fayob , you can search for the font required in [Google font](https://fonts.google.com/). After you have found the font family, choose the font weight required. Then import it and add the font family to your code

Reply:
okay, Thanks